src/speech.cpp

```diff
@@ -13,18 +13,28 @@ static SpeechBubble nullSpeech = { no_translation( "INVALID SPEECH" ), 0 };
 
 void load_speech( JsonObject &jo )
 {
-    const std::string label = jo.get_string( "speaker" );
+    std::vector<std::string> labels;
+    if( jo.has_string( "speaker" ) ) {
```

Comment:
You could use `JsonObject::get_tags`, which will read a single string or an array of strings and return that as `std::set`. 

```C++
for( const std::string &label : jo.get_tags( "speaker" ) ) {
```
(Note as it returns a `std::set`, the order of the array elements may not be kept, if that's important.)

Reply:
Thanks a lot.